metricbeat/module/kubelet/types.go

```diff
@@ -0,0 +1,119 @@
+package kubelet
+
+type Summary struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Summary should have comment or be unexported